[Pytorch Edge] Extend runtime compatibility to custom classes

JacobSzwejbka opened the pull request:
Summary: Add api to view how many custom classes we have and what their names are

Test Plan: unit test

Differential Revision: D31811337

cccclai reviewed (approved):
LGTM after address the minor feedback

JacobSzwejbka committed:
[Pytorch Edge] Extend runtime compatibility to custom classes (#66972)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/66972

Add api to view how many custom classes we have and what their names are

Test Plan: unit test

Reviewed By: cccclai

Differential Revision: D31811337

fbshipit-source-id: d4917813ade2735ee545b6eec45d1fd25912cbf2
